src/js/messaging/reducers/folders.js

```diff
@@ -27,17 +26,11 @@ const initialState = {
       attributes: {},
       filter: {},
       messages: [],
-      pagination: {
-        currentPage: 0,
-        perPage: 0,
-        totalEntries: 0,
-        totalPages: 0
-      },
-      persistFolder: 0,
+      pagination: {},
```

Comment:
That's true. I don't think showing something like "0 of 0" looks that great either, though. Maybe that components that need data that are undefined should be hidden until they're populated?